python/pylibcugraph/pylibcugraph/_cugraph_c/sampling_algorithms.pxd

```diff
@@ -65,6 +65,42 @@ cdef extern from "cugraph_c/sampling_algorithms.h":
         cugraph_error_t** error
     )
 
+    cdef cugraph_error_code_t cugraph_heterogeneous_neighbor_sample(
+        const cugraph_resource_handle_t* handle,
+        cugraph_rng_state_t* rng_state,
+        cugraph_graph_t* graph,
+        const cugraph_edge_property_view_t* edge_biases,
+        const cugraph_type_erased_device_array_view_t* start_vertices,
+        const cugraph_type_erased_device_array_view_t* start_vertex_labels,
+        const cugraph_type_erased_device_array_view_t* label_list,
+        const cugraph_type_erased_device_array_view_t* label_to_comm_rank,
+        const cugraph_type_erased_device_array_view_t* label_offsets,
+        const cugraph_sample_heterogeneous_fan_out_t* heterogeneous_fan_out,
+        const cugraph_sampling_options_t* options,
+        bool_t is_biased,
+        bool_t do_expensive_check,
+        cugraph_sample_result_t** result,
+        cugraph_error_t** error
+    )
+
+    cdef cugraph_error_code_t cugraph_homogeneous_neighbor_sample(
+        const cugraph_resource_handle_t* handle,
+        cugraph_rng_state_t* rng_state,
+        cugraph_graph_t* graph,
+        const cugraph_edge_property_view_t* edge_biases,
+        const cugraph_type_erased_device_array_view_t* start_vertices,
+        const cugraph_type_erased_device_array_view_t* start_vertex_labels,
+        const cugraph_type_erased_device_array_view_t* label_list,
+        const cugraph_type_erased_device_array_view_t* label_to_comm_rank,
+        const cugraph_type_erased_device_array_view_t* label_offsets,
+        const cugraph_type_erased_host_array_view_t* fan_out,
+        const cugraph_sampling_options_t* options,
+        bool_t is_biased,
```

Comment:
Parameter is obsolete.

Reply:
Right. I didn't update the PLC and python API as I was mostly focused on the C++ and CAPI. But these should be address in my next commits